frontend/simple/components/GroupMembers.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <section data-test="groupMembers">
-    <div class="level">
-      <h3 class="title"><i18n>Group Members</i18n></h3>
-    </div>
+  <dashboard-section data-test="groupMembers" title="Group Members">
```

Comment:
Here and in a few other places, see [related comment](https://github.com/okTurtles/group-income-simple/pull/415/files#r191607332) about using `:title="'L(...` instead